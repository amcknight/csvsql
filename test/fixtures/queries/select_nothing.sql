SELECT FROM people WHERE True
