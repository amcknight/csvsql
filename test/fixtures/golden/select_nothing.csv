1:13:
  |
1 | SELECT FROM people WHERE True
  |             ^^^^
unexpected "peop"
expecting "FROM" or white space
